src/components/Filter/FilterInput.vue

```diff
@@ -563,6 +563,7 @@ $input-height: rem(28px);
     z-index: 1;
     // Text indent is needed instead of padding so text inside <input> doesn't get cut off
     text-indent: rem(7px);
+    text-overflow: ellipsis;
```

Comment:
I cannot find a use case when the ellipsis is reproducible. Can you explain a bit more when this would happen?

Reply:
I see that it's for the placeholder :) Looks good then